bundles/org.openhab.persistence.influxdb/src/main/resources/OH-INF/config/config.xml

```diff
@@ -81,6 +81,14 @@
 			<default>false</default>
 		</parameter>
 
+		<parameter name="useMetadataMeasurementName" type="boolean" required="true" groupName="misc">
+			<label>Use metadata name</label>
+			<description>If true uses metadata from item influxdb metadata value as name for measurement instead of item name. If
+				metadata value is not set item name is used.
+			</description>
+			<default>false</default>
+		</parameter>
+
```

Comment:
```suggestion
```
In the review I forgot to say that his must be removed as it isn't any longer used.